ocaml/testsuite/tests/typing-modes/legacy.ml

```diff
@@ -0,0 +1,116 @@
+(* TEST
+   * expect
+*)
+
+(* This file tests various legacy constraints. *)
```

Comment:
```suggestion
(* This file tests various legacy constructs. *)
```

Reply:
(This PR will rebase, where this `legacy.ml` is splitted into `module.ml` `class.ml` etc., and the commented wording doesn't exist. Therefore, I'm resolving.)